.travis.yml

```diff
@@ -17,6 +20,13 @@ cache:
     - app/bower_components
     - fxa-auth-server/node_modules
 
+notifications:
+  irc:
+    channels:
+      - 'irc.mozilla.org#fxa-bots'
+    use_notice: false
+    skip_join: false
+
```

Comment:
I thought it was weird that this repo's .travis.yml file didn't have this section. Is it defined somewhere else?

![](http://i.kinja-img.com/gawker-media/image/upload/japbcvpavbzau9dbuaxf.jpg)